### ISSUE TITLE
Braille viewer doesn't report all formatting

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#14146]
### Steps to reproduce:

1. Turn on the Braille viewer
2. Go into Word
3. Ensure NVDA's document formatting is set to report "Font attributes".
4. Type some text, say "The quick brown fox".
5. Set some of it bold, say the word "brown".
6. Read the line and observe what is displayed in Braille viewer.

### Actual behavior:

Braille viewer displays the correct symbol (dot 4, 5 and dot 2) to indicate the word is in Braille, however the text underneath simply reads "The quick brown fox".

Additionally, if you arrow through the text and the caret is in the middle of the bold word, the bold symbol disappears from the Braille viewer, although it was still reported audibly when focus moved into the word.

### Expected behavior:

I don't have a hardware display handy to text how it actually behaves, however the English text in the braille viewer should accurately represent what the Braille cells are showing.

#### NVDA version:

NVDA 2022.3 Beta 4 installed

#### Windows version:

Windows 11 (64-bit) Version: 21H2 (2009), Build: 22000.978

**Comment by feerrenrut** [comment]
NVDA doesn't have enough information (currently) to be able to do this. The (non-braille) text is there only to give some context about the cells displayed. The main intention was to support developer use-cases. It would be great if we could expand this to support a wider set of use-cases more thoroughly.

**Comment by feerrenrut** [comment]
After the technical challenge of getting a backtranslation for each braille cell (or group of cells). There are some unknowns about the UX:
- For developers, it would be useful to have a raw text character to represent every braille cell.
- A full translated text would be helpful for other use-cases. E.G. expanding contracted braille.
